Makefile

```diff
@@ -190,6 +203,10 @@ envtest: $(ENVTEST) ## Download setup-envtest locally if necessary.
 $(ENVTEST): $(LOCALBIN)
 	$(call go-install-tool,$(ENVTEST),sigs.k8s.io/controller-runtime/tools/setup-envtest,$(ENVTEST_VERSION))
 
+.PHONY: crd-ref-docs
+crd-ref-docs: ## Install crd-ref-docs.
+	GOBIN=$(LOCALBIN) GO111MODULE=on go install github.com/elastic/crd-ref-docs@$(CRD_REF_DOCS_VERSION)
```

Comment:
I see that there is already a [go-install-tool](https://github.com/etcd-io/etcd-operator/blob/340fe746e2261dbec99e1cca34414e4c6c73ad5b/Makefile#L202) defined in the Makefile. We need to have a consistent way to manage all the tools per https://github.com/etcd-io/etcd-operator/pull/13/files#r1898432778. This can be enhanced in followup PRs.

Reply:
Hi, I changed the command to the following:
```
.PHONY: crd-ref-docs
crd-ref-docs: $(CRD_REF_DOCS) ## Install crd-ref-docs tool, ref: https://github.com/elastic/crd-ref-docs.
$(CRD_REF_DOCS): $(LOCALBIN)
	$(call go-install-tool,$(CRD_REF_DOCS),github.com/elastic/crd-ref-docs@$(CRD_REF_DOCS_VERSION))
```